sklearn/metrics/tests/test_pairwise_distances_reduction.py

```diff
@@ -1120,8 +1210,8 @@ def test_radius_neighbors_classmode_factory_method_wrong_usages():
 @pytest.mark.parametrize("dtype", [np.float64, np.float32])
 def test_chunk_size_agnosticism(
     global_random_seed,
-    Dispatcher,
-    dtype,
+    Dispatcher: ArgKmin | RadiusNeighbors,
```

Comment:
Please remove all type annotations and spurious formating changes which have been introduced in this file (and elsewhere if any).